channels/views.py

```diff
@@ -312,9 +311,54 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
             )
 
 
+class MoreCommentsView(APIView):
+    """
+    View for expanding a MoreComments object
+    """
+
+    def get_serializer_context(self):
+        """Context for the request and view"""
+        return {
+            'request': self.request,
+            'view': self,
+        }
+
+    def get(self, request, *args, **kwargs):  # pylint: disable=unused-argument
+        """Get list for comments and attach User objects to them"""
+        with translate_praw_exceptions():
+            children = request.query_params.getlist('children')
+
+            # validate the request parameters: each are required
+            try:
+                post_id = request.query_params['post_id']
+                parent_id = request.query_params['parent_id'] or None
```

Comment:
Is an empty string expected here or is the `or None` extraneous?

Reply:
There's no use case for an empty string, I'll see about removing it from the frontend if it's null